### DISCUSSION TITLE
Nội dung bài Word

### DISCUSSION THREAD

**Discussion by PhamThanhPhongPN** [PhamThanhPhongPN/Lop-10A02-Nhom-07#29] [Ideas]
Phân công và chọn nội dung làm

**Comment by PhamThanhPhongPN** [comment]
![image](https://user-images.githubusercontent.com/92080858/155907285-57e8e295-a297-4a2b-9fc2-2bc2a2b36021.png)

**Reply by PhamThanhPhongPN** [reply]
![image](https://user-images.githubusercontent.com/92080858/155907306-b2efe097-5bc5-4a86-b1ae-3dc10f9f4c22.png)

**Comment by PhamThanhPhongPN** [comment]
alo =)))

**Comment by PhamThanhPhongPN** [comment]
vậy theo đề là chọn 3 câu thành ngữ/ tục ngữ/ ca dao

**Comment by PhamThanhPhongPN** [comment]
mà tuần 1 bên mình chưa phân công gì hết

**Comment by PhamThanhPhongPN** [comment]
đầu tiên là phải chọn 3 cái
https://thptsoctrang.edu.vn/tong-hop-cac-cau-ca-dao-tuc-ngu-viet-nam-day-du-nhat/
vô đây chọn luôn nha cho nó dễ

**Reply by Nguyenhominhdang10A2** [reply]
Ok

**Comment by PhamThanhPhongPN** [comment]
xong rồi tới phần phân công. Mỗi tuần làm 1 câu cho dễ
Mỗi câu thì cần:
3 bạn tìm thông tin/hình ảnh
1 bạn tổng hợp thông tin, viết bài
1 bạn làm word với trang trí

**Reply by PhamThanhPhongPN** [reply]
cho t xin làm word

**Reply by PhamThanhPhongPN** [reply]
mốt qua câu khác thì đổi nhiệm vụ

**Reply by Nguyenhominhdang10A2** [reply]
Ok

**Comment by PhamThanhPhongPN** [comment]
tuần này mình chọn câu: "Đi một ngày đàng học một sàng khôn." nha

**Reply by Nguyenhominhdang10A2** [reply]
Ok

**Comment by PhamThanhPhongPN** [comment]
ai muốn làm gì thì nhắn vô đây

**Reply by PhamThanhPhongPN** [reply]
hạn là trc t4 tuần sau

**Reply by Nguyenhominhdang10A2** [reply]
tuần này co ai làm trc hộ t đc kh??

**Reply by Nguyenhominhdang10A2** [reply]
Tuần sau t làm lai cho

**Reply by PhamThanhPhongPN** [reply]
vậy tuần này để t viết luôn cho

**Reply by PhamThanhPhongPN** [reply]
vậy @Huynhgiahao10A02 @Nguyen-Hoang-Bao-Dung @TuyenTuyenTuyen  tìm thông tin nha

**Reply by PhamThanhPhongPN** [reply]
1.Lời mở đầu / dẫn dắt
+ Nêu vấn đề, khái quát ý nghĩa
2. Giải thích
+ Nghĩa đen
+ Nghĩa bóng
=> Bài học / nhận xét / lời khuyên từ câu trên
3. Chứng minh
+ Tại sao cần ... ? / Ý nghĩa
+ Dẫn chứng ( thực tế, mới lạ )
4. Bài học rút ra
+ Suy nghĩ của bản thân về câu được chọn
+ Bài học từ câu ...
+ Kêu gọi, khuyến khích mọi người ...
5. Mở rộng vấn đề
+ Lên án / phê phán những ai ...
6. Khẳng định lại về giá trị
+ Nhận thức, hành động của bản thân

**Reply by PhamThanhPhongPN** [reply]
mấy bạn tìm thông tin thì tìm giống mấy phần trên nha

**Reply by HaoHuynh312006** [reply]
OK ok

**Reply by TheTuyenTuyen** [reply]
từ bé đến lớn giờ chưa thấy cái nào ảo dư lày

**Comment by TheTuyenTuyen** [comment]
rồi t làm gì giờ
